Solution4.py

```diff
@@ -14,7 +14,7 @@
 with open("C:\sqlite3\salary.csv","r") as read_obj:
     csv_reader=csv.reader(read_obj)
     for row in csv_reader:
-        if row[1][0]=='s':      #if employee name starts with 's',it will pass
+        if row[1][0]=='s' or row[1][0]=='S':      #if employee name starts with 's',it will pass
```

Comment:
how it works?

Reply:
row contains records which were originally in csv file and check condition using if statement, if the name starts with "S" or "s", the condition will be true and the record will be printed